Sources_v3/StreamChatUI/Generated/L10n.swift

```diff
@@ -0,0 +1,74 @@
+// swiftlint:disable all
```

Comment:
Is this line part of the generated file or you added it? If the latter, it will get removed the next time the file gets regenerated.

Reply:
The generated one